test/testallfilesdeleted.cpp

```diff
@@ -70,8 +70,10 @@ private slots:
             });
 
         auto &modifier = deleteOnRemote ? fakeFolder.remoteModifier() : fakeFolder.localModifier();
-        for (const auto &s : fakeFolder.currentRemoteState().children.keys())
+        const auto childrenKeys = fakeFolder.currentRemoteState().children.keys();
+        for (const auto &s : childrenKeys) {
```

Comment:
this is also a single letter variable

Reply:
I agree with Claudio, especially on this. `s` is misleading, it does not stand for `childrenKeys`. Something like `chKey` would be much more readable.